tempest/src/test/kotlin/app/cash/tempest/LogicalDbTransactionTest.kt

```diff
@@ -251,9 +253,10 @@ class LogicalDbTransactionTest {
       )
       .build()
 
-    assertFailsWith<TransactionCanceledException> {
+    val exception = assertThrows<AmazonServiceException> {
```

Comment:
Unfortunate consequence of a bug in the DynamoDb Local client where it throws a different exception than what would be thrown in a real-world scenario.